scripts/apps/authoring/authoring/directives/AuthoringTopbarDirective.ts

```diff
@@ -45,13 +46,34 @@ export function AuthoringTopbarDirective(
             };
 
             scope.openPublishOrSendToPane = () => {
+                const availableTabs = getAvailableTabs();
+                const activeTab = getActiveTab(availableTabs);
+
                 dispatchInternalEvent('interactiveArticleActionStart', {
                     items: [scope.item],
-                    tabs: ['send_to', 'publish'],
-                    activeTab: scope.item.flags?.marked_for_not_publication ? 'send_to' : 'publish',
+                    tabs: availableTabs,
+                    activeTab: activeTab,
                 });
             };
 
+            function getAvailableTabs(): Array<IArticleActionInteractive> {
```

Comment:
Integrate requirements from SDESK-7079  (#4370) to this function, because now you removed it completely.